payments/reseller/subscription/snippets/src/main/java/com/google/cloud/reseller/subscription/samples/Constants.java

```diff
@@ -0,0 +1,17 @@
+package com.google.cloud.reseller.subscription.samples;
+
+public class Constants {
+
+  public static final String PARTNER_ID = "yt_ptm_staging";
+  public static final String NFT_PRODUCT_ID = "YTPD-acb26288-d14b-4865-a748-a132c5f37f6d";
+  public static final String PRODUCT_ID = "YTPD-902fd916-219d-4aa0-b735-40fc57bd9b5b";
+  public static final String ELIGIBILITY_ID = "yt_ptm_staging:ibv2-bundle-plan";
+  public static final String PROMOTION = "YTPM-CilZVFBELTkwMmZkOTE2LTIxOWQtNGFhMC1iNzM1LTQwZmM1N2JkOWI1YhAC";
+  public static final String PROMOTION_SUB_LEVEL = "YTPM-CilZVFBELTkwMmZkOTE2LTIxOWQtNGFhMC1iNzM1LTQwZmM1N2JkOWI1YhAB";
+  public static final String PLAN_TYPE = "PARTNER_PLAN_TYPE_HARD_BUNDLE";
+  public static final String REGION_CODE = "US";
+  public static final String SUB_ID = "onboarding-partner-testing";
+  public static final String TARGET_SERVICE_ACCOUNT_EMAIL = "payment-reseller-test-serv-734@eng-diagram-369411.iam.gserviceaccount.com";
```

Comment:
If you need environment variables, these should be setup in `run_tests.sh` ((docs)](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#testing-setup)). 
However, these values appear to be extremely specific, possibly for the project in the service account email. 

We will need to either make sure the CI testing account has this API and values appropriate for the account, or setup a dedicated project that has access to these values (we typically do not assume a provided project will support ongoing CI testing without other communication)